main.go

```diff
@@ -156,6 +152,14 @@ func getCommands() []cli.Command {
 				return commands.RunCiSetupCmd()
 			},
 		},
+		{
+			Name:        cliutils.CmdEnvironments,
+			Aliases:     []string{"envs"},
+			Description: "Shows a list of environment variables",
```

Comment:
Shows a list of environment variables --> Show all supported environment variables